Dockerfile.alpine

```diff
@@ -0,0 +1,31 @@
+FROM nginx:1.11.8-alpine
+MAINTAINER Jason Wilder mail@jasonwilder.com
+
+# Install wget and install/updates certificates
+RUN apk add --no-cache --virtual .run-deps \
+    ca-certificates bash wget \
+    && update-ca-certificates
+
+# Configure Nginx and apply fix for very long server names
+RUN echo "daemon off;" >> /etc/nginx/nginx.conf \
+ && sed -i 's/^http {/&\n    server_names_hash_bucket_size 128;/g' /etc/nginx/nginx.conf
+
+# Install Forego
+ADD https://github.com/jwilder/forego/releases/download/v0.16.1/forego /usr/local/bin/forego
```

Comment:
I guess the nginx:alpine image installs some necessary libs. libc-dev? https://github.com/nginxinc/docker-nginx/blob/master/stable/alpine/Dockerfile#L55
I don't know.